docs/bundle/configuration.md

```diff
@@ -9,7 +9,7 @@ Then configure the bundle to your needs, for example:
 ```yaml
 automapper:
   class_prefix: "Symfony_Mapper_"
-  allow_constructor: true
+  constructor_strategy: 'auto'
```

Comment:
shouldn't we allow to configure this at runtime, thanks to the context?

Reply:
I think having this at runtime would loose performance over a "niche" feature, my goal with that PR is more : I want to generate mapping with the same consistent behavior (never use constructor, always use it, try to use it if possible)

Having this at runtime will complicate thing IMO, let's wait for someone who need that to make this change if needed ?